example/cycle/test/test.js

```diff
@@ -7,31 +7,25 @@ import * as actions from '../../actions';
 
 import { fetchReposByUser, searchUsers } from '../';
 
-function assertSourcesSinks(sources, sinks, main, done) {
-  const Time = mockTimeSource();
+function assertSourcesSinks(sources, sinks, main, done, timeOpts = {}) {
```

Comment:
https://gist.github.com/lmatteis/b1d18f863f589c496dc604b95ca56454#file-test-js-L18-L23

Doesn't seem to specify the selector or event name, which would be necessary to distinguish between. There is also `isolation` to consider, the implementation of which varies driver to driver.

I wonder if you could integrate this with `mockDOMSource` somehow.